pkg/op-data/op_data/sources/discourse/topics.py

```diff
@@ -111,5 +111,14 @@ async def acquire_and_save():
 
     @staticmethod
     async def update_relationships():
-        # If there are any relationships to update, implement them here
-        pass
+        topics = await Topic.all().prefetch_related('rawTopic')
+
+        for topic in topics:
+            raw_related_topics = topic.rawTopic.rawData.get("related_topics", None)
+
+            if isinstance(raw_related_topics, list):
```

Comment:
could it be anything other than a list or None?

Reply:
I don't have sure